.env.sample

```diff
@@ -14,8 +17,17 @@ MAGICC_EXECUTABLE_7=/path/to/magicc/bin/magicc
 MAGICC_WORKER_NUMBER=4
 
 # Where should the MAGICC workers be located on the filesystem (you need about
-# 500Mb space per worker at the moment)
+# 500Mb space per worker at the moment)?
 MAGICC_WORKER_ROOT_DIR=~/Desktop
 
+### Cicero-SCM ###
+# ------------- #
+CICEROSCM_WORKER_NUMBER=4
+
+# Where should Cicero-SCM workers be located on the filesystem
```

Comment:
Note to myself to add this

Reply:
Turns out all that was needed one was extra line: https://github.com/maritsandstad/openscm-runner/pull/7